src/ImageSharp/Image/ImageFrame{TPixel}.cs

```diff
@@ -168,7 +168,9 @@ internal void SwapPixelsBuffers(ImageFrame<TPixel> pixelSource)
         {
             Guard.NotNull(pixelSource, nameof(pixelSource));
 
-            ComparableExtensions.Swap(ref this.pixelBuffer, ref pixelSource.pixelBuffer);
+            Buffer2D<TPixel> temp = this.pixelBuffer;
```

Comment:
Suggest var.

Reply:
Why? In the statement you cannot see from the right hand side what the type is. Your suggestion deliberately obfuscates the codebase.